encryption/caesar.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace encryption {
+
+    class Caesar {
```

Comment:
Let's change such approach in little bit different way :
1) Add interface for Encryption/Decryption
2) Add inheritance implementation for classical Caesar 
3) Add inheritance implementation for modern Caesar